client/state/posts/utils.js

```diff
@@ -249,3 +251,12 @@ export function normalizePostForApi( post ) {
 
 	return normalizeApiFlow( post );
 }
+
+/**
+ * Check if the post content is empty (ignoring empty tags)
+ * @param  {string}  content Raw post content
+ * @return {Boolean}         Whether it's considered empty
+ */
+export function isPostContentEmpty( content ) {
+	return ! content || ( content.length < CONTENT_LENGTH_ASSUME_SET && REGEXP_EMPTY_CONTENT.test( content ) );
+}
```

Comment:
this is named `isPost ContentEmpty` and yet it doesn't really have much to do with any `post`. It works completely independently. What if we just called it something like, `isEmpty` or `isEmptyContent` or `isBlank`